test/integration/goldens/logging/LoggingServiceV2Client.java

```diff
@@ -435,7 +435,13 @@ public final WriteLogEntriesResponse writeLogEntries(WriteLogEntriesRequest requ
    *
    * <pre>{@code
    * try (LoggingServiceV2Client loggingServiceV2Client = LoggingServiceV2Client.create()) {
-   *   // Note: Not implemented yet, placeholder for paged unary rpc method sample code.
+   *   List<String> resource_names = new ArrayList<>();
+   *   String filter = "filter-1274492040";
+   *   String order_by = "order_by1234304744";
+   *   for (LoggingServiceV2Client loggingServiceV2Client :
```

Comment:
PTAL at the [sample](https://github.com/googleapis/java-logging/blob/master/google-cloud-logging/src/main/java/com/google/cloud/logging/v2/LoggingClient.java#L292), the for-loop variable is not the client.